tests/DataSet/ObjectDataSetTest.php

```diff
@@ -268,6 +332,70 @@ public function testValidateWithWrongCallbackMethod(): void
                 $object::class,
             )
         );
-        new ObjectDataSet($object);
+        (new ObjectDataSet($object))->getRules();
```

Comment:
We testing `getRules()`. `ObjectDataSet` should be created before asserts.

Reply:
It's already fixed. Maybe diff was shown incorrectly.